db/schema.rb

```diff
@@ -82,8 +90,8 @@
     t.text "full_time_part_time"
     t.boolean "teaching_scholars"
     t.uuid "dttp_id"
-    t.integer "record_type"
     t.text "middle_names"
+    t.integer "record_type"
```

Comment:
I've noticed for some odd reason this attribute seems to change order for some reason after a db migrate